third_party/prism.BUILD

```diff
@@ -0,0 +1,61 @@
+GENERATED_SRCS = [
+  "src/diagnostic.c",
+  "src/node.c",
+  "src/prettyprint.c",
+  "src/serialize.c",
+  "src/token_type.c",
+]
+
+GENERATED_HDRS = [
+  "include/prism/ast.h",
+  "include/prism/diagnostic.h",
+]
+
+genrule(
+  name = "generate_templates",
+  srcs = [
+    "templates/template.rb",
+    "prism.gemspec",
+    "Gemfile",
+    "Gemfile.lock",
+    "config.yml", # Contains the data to populate the ERB templates.
+  ] + \
+  ["templates/{c_file}.erb".format(c_file = c_file) for c_file in GENERATED_SRCS] + \
+  ["templates/{h_file}.erb".format(h_file = h_file) for h_file in GENERATED_HDRS],
+  outs = GENERATED_HDRS + GENERATED_SRCS,
+  cmd = """
```

Comment:
This isn't great, but it's a good-enough workaround for now.

Ideally we'd find a way to conveniently use whatever Ruby version the user already has (above some minimum required version)

We very much welcome ideas for how to improve this. We didn't want to get blocked early on fighting with Bazel, so we're working off this for now